indexes/clock.js

```diff
@@ -38,9 +38,9 @@ module.exports = function (db, opts) {
           sync: false === (opts && opts.sync),
           limit: limit
         }),
-        //NEVER allow private messages over history stream.
-        //createHistoryStream is used for legacy replication.
-        u.Format(keys, values, false)
+        // NEVER allow private messages over history stream.
+        // createHistoryStream is used for legacy replication.
+        u.Format(keys, values, true)
```

Comment:
@dominictarr This doesn't change the behavior of `createHistoryStream`, which continues to output *only* the original message, but I don't mind going back to checking for `opts.private` if that's the most ergonomic.